alpha/declcfg/declcfg_to_model_test.go

```diff
@@ -291,7 +440,7 @@ func TestConvertToModel(t *testing.T) {
 }
 
 func TestConvertToModelRoundtrip(t *testing.T) {
-	expected := buildValidDeclarativeConfig(true)
+	expected := buildValidDeclarativeConfig(validDeclarativeConfigSpec{IncludeUnrecognized: true, IncludeDeprecations: false}) // TODO: turn on deprecation when we have model-->declcfg conversion
```

Comment:
This function goes declcfg --> model --> declcfg and compares the start and finish products.  We don't yet have backwards conversion, so this can't be enabled until #1162 lands.